.github/workflows/poc-storage.yml

```diff
@@ -73,16 +74,41 @@ jobs:
       id: meta
       uses: docker/metadata-action@69f6fc9d46f2f8bf0d5491e4aabe0bb8c6a4678a
       with:
+        sep-tags: ","
+        sep-labels: ","
         images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+        # Docker tags based on the following events/attributes
+        tags: |
+          type=schedule
+          type=ref,event=branch
+          type=ref,event=pr
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
+          type=semver,pattern={{major}}
+          type=sha
 
-    # Build and push Docker image with Buildx (don't push on PR)
+    # Build and push Docker image with Buildx
     # https://github.com/docker/build-push-action
     - name: Build and push Docker image
       id: build-and-push
+      if: ${{ github.event_name != 'pull_request' }}
       uses: docker/build-push-action@1cb9d22b932e4832bb29793b7777ec860fc1cde0
       with:
         context: storage
-        push: ${{ github.event_name != 'pull_request' }}
+        push: true
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
+
+    # Build and push Docker image with Buildx
+    # https://github.com/docker/build-push-action
```

Comment:
fix comment, say something about pushing this on every PR with different tag and why